src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@keyframes rotates {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  inset: 0;
+  margin: auto;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minute-hand {
+    margin: auto;
+    position: absolute;
+    left: inherit;
+    right: 0;
```

Comment:
You have used `left: inherit;` and `right: 0;`. The `left` property is not necessary here, as it's not adding any value to your code. Please remove it.